neo/ProtocolSettings.cs

```diff
@@ -88,6 +89,7 @@ private ProtocolSettings(IConfigurationSection section)
             }
             this.SystemFee = sys_fee;
             this.SecondsPerBlock = section.GetValue("SecondsPerBlock", 15u);
+            this.StateRootEnableIndex = section.GetValue("StateRootEnableIndex", 0u);
```

Comment:
We should define by default the expected one

Reply:
OK

Reply:
Of course, only don't forget it.

Reply:
BTW, We have `StateRootEnableIndex` setting in `protocol.json`.
Besides, Height of `mainnet` is different from `testnet`. 
I don't think it's necessary to focus on this default value.